docs/recipes-text.rst

```diff
@@ -380,89 +380,189 @@ How to Fill a Text Box
+The method is based on the :ref:`Story` class. Therefore, complex script systems like Devanagari, Nepali, Tamil and many are supported and written correctly thanks to using the HarfBuzz library - which provides this feature, called *"text shaping"*.
 
-    page.insert_text(point, russian_text, encoding=fitz.TEXT_ENCODING_CYRILLIC)
+Any required fonts to output characters are automatically pulled in from the Google NOTO font library - as a fallback when the optionally supplied user font(s) do not contain some glyphs.
 
-The valid encoding values are TEXT_ENCODING_LATIN (0), TEXT_ENCODING_GREEK (1), and TEXT_ENCODING_CYRILLIC (2, Russian) with Latin being the default. Encoding can be specified by all relevant font and text insertion methods.
+As a small glimpse into the features offered here, we will output the following HTML-enriched text::
 
-By the above statement, the fontname *helv* is automatically connected to the Russian font variant of Helvetica. Any subsequent text insertion with **this fontname** will use the Russian Helvetica encoding.
+    import fitz
 
-If you change the fontname just slightly, you can also achieve an **encoding "mixture"** for the **same base font** on the same page::
 
-    import fitz
-    doc=fitz.open()
+    rect = fitz.Rect(100, 100, 400, 300)
+
+    text = """Lorem ipsum dolor sit amet, consectetur adipisici elit, sed
+        eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation <b>ullamco <i>laboris</i></b> 
+        nisi ut aliquid ex ea commodi consequat. Quis aute iure 
+        <span style="color: #f00;">reprehenderit</span> 
+        in <span style="color: #0f0;font-weight:bold;">voluptate</span> velit 
+        esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat 
+        cupiditat non proident, sunt in culpa qui 
+        <a href="https://www.artifex.com">officia</a> deserunt mollit anim id 
+        est laborum."""
+
+    doc = fitz.Document()
+
     page = doc.new_page()
-    shape = page.new_shape()
-    t="Sômé tèxt wìth nöñ-Lâtîn characterß."
-    shape.insert_text((50,70), t, fontname="helv", encoding=fitz.TEXT_ENCODING_LATIN)
-    shape.insert_text((50,90), t, fontname="HElv", encoding=fitz.TEXT_ENCODING_GREEK)
-    shape.insert_text((50,110), t, fontname="HELV", encoding=fitz.TEXT_ENCODING_CYRILLIC)
-    shape.commit()
-    doc.save("t.pdf")
+    page.insert_htmlbox(rect, text, css="* {font-family: sans-serif;font-size:14px;}")
+
+    doc.ez_save(__file__.replace(".py", ".pdf"))
+
+Please note how the "css" parameter is used to globally select the default "sans-serif" font and a font size of 14.
+
+The result will look like this:
+
+.. image:: images/img-htmlbox1.*
+   :scale: 30
+
+Here is another example that outputs a table with this method. This time, we are including all the styling in the HTML source itself. Please also not, how it works to also include an image - even within a table cell::
```

Comment:
typo: not -> note
Also I don't think it needs the comma after "Please also note ..." as it is all one sentence flow.